login-signup/pages/signup/index.js

```diff
@@ -0,0 +1,101 @@
+(() => {
+  const validateName = (name) => {
+    const re = /^([a-zA-Z]{1,4}\.)?([ a-zA-Z]{4,20})$/;
```

Comment:
Define it as constant as follows
`const NAME_REGEX = /^([a-zA-Z]{1,4}\.)?([ a-zA-Z]{4,20})$/;`

Reply:
Defined as instructed